docs/en/query_language/functions/hash_functions.md

```diff
@@ -66,9 +143,30 @@ Levels are the same as in URLHierarchy. This function is specific to Yandex.Metr
 
 ## farmHash64
 
-Calculates FarmHash64 from a string.
-Accepts a String-type argument. Returns UInt64.
-For more information, see the link: [FarmHash64](https://github.com/google/farmhash)
+Produces a 64-bit [FarmHash](https://github.com/google/farmhash) hash value.
```

Comment:
There are multiple variants and we'd better document which exactly it is (as far as I remember non-fingerprint one)
https://github.com/google/farmhash/blob/master/src/farmhash.h